Description: fix build failure with gcc5:
 build with "-std=gnu89"
Origin: vendor
Bug-Debian: https://bugs.debian.org/778171
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2015-07-19

--- a/wmmon/Makefile
+++ b/wmmon/Makefile
@@ -5,7 +5,7 @@
 		../wmgeneral/misc.o \
 		../wmgeneral/list.o
 
-CFLAGS = -O2
+CFLAGS = -O2 -std=gnu89
 CC = cc
 
 
